### PR TITLE
Add API Endpoint: List Services for Org

### PR DESCRIPTION
**Pull request by moritzraho**
## Description
the remaining endpoints will be added later

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: spec/apinew.json [hunk 1]
```diff
@@ -0,0 +1,7448 @@
+{
```
Comment (Himavanth) [inline spec/apinew.json:1]: What is this file and the other apiold.json for ?

File: spec/api.json [hunk 2]
```diff
@@ -4818,6 +4818,141 @@
           ]
         }
       },
+      "/organizations/{orgId}/services": {
+        "get": {
+          "summary": "list services for an organization",
+          "tags": [
+            "Organizations"
+          ],
+          "operationId": "getServicesForOrg",
```
Comment (Himavanth) [inline spec/api.json:4827]: This is good. I was looking for consistency and noticed the `getProjectsByOrId` should be `getProjectsForOrg`. Not related to your changes and not a serious requirement though.

**Review by Himavanth — APPROVED** [review]
LGTM.

File: spec/apinew.json (hunk 1 above)
Comment (moritzraho) [reply spec/apinew.json]: oops deleted :)

File: spec/api.json (hunk 2 above)
Comment (moritzraho) [reply spec/api.json:4827]: ok added a change

File: src/index.js [hunk 3]
```diff
@@ -573,6 +573,26 @@ class CoreConsoleAPI {
...
   }
 
+  /**
+   * Get all Services available to an Organization
+   *
+   * @param {string} organizationId Organization ID
+   * @returns {Promise<Response>} the response
+   */
+  getServicesForOrg (organizationId) {
+    const sdkDetails = {}
```
Comment (shazron) [inline src/index.js:583]: sdkDetails should contain the organizationId for debugging purposes

**Review by shazron — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 3 above)
Comment (moritzraho) [reply src/index.js:583]: ok good point thanks

**Pull request merged**